### PR TITLE
formatted docstring for bitwise_xor in ivy backend

### PR DESCRIPTION
**Pull request by abundance-io**
- [ ] #761

### CODE REVIEW

File: ivy/functional/ivy/elementwise.py [hunk 1]
```diff
@@ -266,25 +266,182 @@ def bitwise_xor(
...
+
+    an array containing the element-wise results. The returned array must have a
+
+    data type determined by :ref:`type-promotion`.
+
+
+    This method conforms to the `Array API Standard
+
+    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.add.html>`_ # noqa
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:311]: Update URL

**Review by chie2727 — CHANGES_REQUESTED** [review]
_Changes requested_

**Review by chie2727 — CHANGES_REQUESTED** [review]
Hi @abundance-io ,
Thanks for making the changes.
You need one more functional example in the `With :code: 'ivy.Array' input` section that updates the input array by using either `out=a` or `out=b`.

See Line 142 for reference: https://github.com/unifyai/ivy/blob/b0832f862245b12f9338442df696c058cb450ab1/ivy/functional/ivy/elementwise.py#L142

File: ivy/functional/ivy/elementwise.py [hunk 2]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    and also the examples below.
+
+
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+
+    >>> a = ivy.array([1,2,3])
+    >>> b = ivy.array([3,2,1])
+    >>> y = ivy.bitwise_xor(a,b)
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:328]: `a = ivy.array([1,2,3])` -> `a = ivy.array([1, 2, 3])`
`b = ivy.array([3,2,1])` -> `b = ivy.array([3, 2, 1])`
`y = ivy.bitwise_xor(a,b)` -> `y = ivy.bitwise_xor(a, b)`

File: ivy/functional/ivy/elementwise.py [hunk 3]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> print(y)
+        ivy.array([2, 0, 2])
+        
+        
+    >>> a = ivy.array([78,91,23])
+    >>> b = ivy.array([66,77,88])
+    >>> ivy.bitwise_xor(a,b,out=y)
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:335]: `a = ivy.array([78,91,23])` -> `a = ivy.array([78, 91, 23])`
`b = ivy.array([66,77,88])` -> `b = ivy.array([66, 77, 88])`
`ivy.bitwise_xor(a,b,out=y)` -> `ivy.bitwise_xor(a, b, out=y)`

File: ivy/functional/ivy/elementwise.py [hunk 4]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> print(y)
+        ivy.array([12, 22, 79])
+
+    >>> a = ivy.array([1,2,3])
+    >>> b = ivy.array([3,2,1])
+    >>> ivy.bitwise_xor(a,b,out = a)
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:341]: -> `a = ivy.array([1, 2, 3])`
-> `b = ivy.array([3, 2, 1])`
-> `ivy.bitwise_xor(a, b, out=a)`

File: ivy/functional/ivy/elementwise.py [hunk 5]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+        ivy.array([2, 0, 2])
+        
+        
+    With :code: `ivy.NativeArray` input:
+
+    >>> a = ivy.native_array([0,1,3,67,91])
+    >>> b = ivy.native_array([4,7,90,89,98])
+    >>> y = ivy.bitwise_xor(a,b)
+    >>> print(y)
+        ivy.array([[ 4,  6, 89, 26, 57]])
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:352]: -> `([0, 1, 3, 67, 91])`
-> `([4, 7, 90, 89, 98])`
-> `(a, b)`

-> `([[4, 6, 89, 26, 57]])`

File: ivy/functional/ivy/elementwise.py [hunk 6]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+
+    >>> a = ivy.array([1,2,3])
+    >>> b = ivy.array([3,2,1])
+    >>> y = ivy.bitwise_xor(a,b)
+    >>> print(y)
+        ivy.array([2, 0, 2])
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:331]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 7]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    `docstring <https://data-apis.org/array-api/latest/API_specification/generated/signatures.elementwise_functions.bitwise_xor.html>`_ # noqa
+
+    in the standard. The descriptions above assume an array input for simplicity, but
+
+    the method also accepts :code:`ivy.Container` instances in place of
+
+    :code:`ivy.Array` or :code:`ivy.NativeArray` instances, as shown in the type hints
+
+    and also the examples below.
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:320]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 8]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> ivy.bitwise_xor(a,b,out=y)
+    >>> print(y)
+        ivy.array([12, 22, 79])
+
+    >>> a = ivy.array([1,2,3])
+    >>> b = ivy.array([3,2,1])
+    >>> ivy.bitwise_xor(a,b,out = a)
+    >>> print(a)
+        ivy.array([2, 0, 2])
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:344]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 9]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    With :code: `ivy.NativeArray` input:
+
+    >>> a = ivy.native_array([0,1,3,67,91])
+    >>> b = ivy.native_array([4,7,90,89,98])
+    >>> y = ivy.bitwise_xor(a,b)
+    >>> print(y)
+        ivy.array([[ 4,  6, 89, 26, 57]])
+
+    
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:355]: Delete these 2 line-spaces

File: ivy/functional/ivy/elementwise.py [hunk 10]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    With a mix of :code: `ivy.Array` and :code:`ivy.NativeArray` inputs:
+    >>> a = ivy.array([0,1,3,67,91])
+    >>> a = ivy.native_array([4,7,90,89,98])
+    >>> y = ivy.bitwise_xor(a,b)
+    >>> print(y)
+        ivy.array([[ 4,  6, 89, 26, 57]])
+        
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:363]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 11]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> y = ivy.array([12,13]])
+
+    >>> z = ivy.bitwise_xor(x,y)
+    >>> print(z)
+    {
+        a: ivy.array([-79,24])
+        b: ivy.array([6,74])
+    }
+        
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:393]: Delete these 2 line-spaces

File: ivy/functional/ivy/elementwise.py [hunk 12]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+
+    Instance Method Examples
+    ------------------------
+    Using :code: `ivy.Array` instance method:
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:397]: Add a line-space above here

File: ivy/functional/ivy/elementwise.py [hunk 13]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+
+    >>> a = ivy.array([[89,51,32],[14,18,19]])
+    >>> b = ivy.array([[[19,26,27],[22,23,20]]])
+    >>> y = a.bitwise_xor(b)
+    >>> print(y)
+    ivy.array([[74,41,59],
+                [24,5,7]])
+
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:406]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 14]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+                        b = ivy.array([78]))
+                        
+    >>>z = x.bitwise_xor(y)
+    >>>print(z)
+    {
+    a:ivy.array([20])
+    b:ivy.array([85])
+    }
+
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:421]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 15]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    Operator Examples
+    -----------------
+    With :code:`ivy.Array` instances:
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:424]: Add a line-space above here

File: ivy/functional/ivy/elementwise.py [hunk 16]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> a = ivy.array([1,2,3])
+    >>> b = ivy.array([3,2,1])
+    >>> y = a ^ b 
+    >>> print(y)
+        ivy.array([2, 0, 2])
+        
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:431]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 17]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+                        b = ivy.array([78]))
+                        
+    >>> z = x ^ y 
+    >>> print(z)
+    {
+    a:ivy.array([20])
+    b:ivy.array([85])
+    }
+
+
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:447]: Delete this line-space

File: ivy/functional/ivy/elementwise.py [hunk 18]
```diff
@@ -266,25 +266,197 @@ def bitwise_xor(
...
+    >>> a = ivy.native_array([0,1,3,67,91])
+    >>> b = ivy.native_array([4,7,90,89,98])
+    >>> y = ivy.bitwise_xor(a,b)
+    >>> print(y)
+        ivy.array([[ 4,  6, 89, 26, 57]])
+
+    
+
+    With a mix of :code: `ivy.Array` and :code:`ivy.NativeArray` inputs:
+    >>> a = ivy.array([0,1,3,67,91])
```
Comment (chie2727) [inline ivy/functional/ivy/elementwise.py:357]: Go through all the remaining arrays and correct the spacing for them too!

**Review by chie2727 — CHANGES_REQUESTED** [review]
Just need to make a few edits to the formatting and you'll be done

**Comment by chie2727** [comment]
Hi @abundance-io , let me know when you've finished making edits to the line-spacing and array spacing.

**Comment by abundance-io** [comment]
I've corrected all the lines you mentioned 
and corrected the spacing for the arrays 

i'm not sure what's left 
is there something i'm missing?

**Comment by chie2727** [comment]
@abundance-io 
I was referring to any remaining double-line spaces as I cannot mention them all. Please change the rest to single-line spaces.

As for the array spacing, for numbers you have typed as `3,4,5` and `a,b,out = a` for example, simply change the spacing to `3, 4, 5` and `a, b, out=a`.
Very minor changes, but it's ideal to keep the formats all consistent.

**Comment by abundance-io** [comment]
I'm done correcting everything 
sorry for taking so long

**Pull request merged**